Dockerfile

```diff
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /build
+
+COPY . /build/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        autoconf \
+        automake \
+        bzip2-dev \
+        curl-dev \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl \
+        openssl-dev \
+        pcre2-dev \
+        zlib-dev \
+    && \
+    rm -rf "/var/cache/apk/"* && \
+    autoreconf --force --install --verbose && \
+    LIBS=-lfts ./configure \
+                           --bindir="/usr/bin" \
+                           --disable-llvm \
+                           --enable-bigstack \
+                           --enable-clamdtop \
+                           --enable-clamonacc \
+                           --enable-milter \
+                           --infodir="/usr/share/info" \
+                           --libdir="/usr/lib" \
+                           --libexecdir="/usr/libexec" \
+                           --localstatedir="/var/lib/clamav" \
+                           --mandir="/usr/share/man" \
+                           --prefix="/usr" \
+                           --sbindir="/usr/sbin" \
+                           --sysconfdir="/etc/clamav" \
+                           --with-dbdir="/var/lib/clamav" \
+                           --with-group="clamav" \
+                           --with-libcurl \
+                           --with-libjson \
+                           --with-openssl="/usr" \
+                           --with-pcre \
+                           --with-user="clamav" \
+                           --with-xml \
+                           --with-zlib \
+                           --without-iconv \
```

Comment:
iconv is required in clamav to convert from different text encodings to utf-8.  I recommend adding it as a dependency.  I think it is actually required for non-Windows builds with CMake.

Reply:
right, again, this is how alpine did it; but i'll do this improvement in combination with cmake

Reply:
with cmake, this seems to be no problem and iconv is included and enabled again